include/pmacc/pluginSystem/TimeSlice.hpp

```diff
@@ -74,6 +74,22 @@ namespace pmacc
                 values({0, uint32_t(-1), 1})
             {
             }
+
+            /** Constructor
+             *
+             * Select any time step in the range [start,end].
+             *
+             * @param start first time step taken into account
+             * @param end last time step taken into account
+             * @param period select any period(st) time step between start and end
+             */
+            TimeSlice(uint32_t start, uint32_t end = uint32_t(-1), uint32_t period = 1)
```

Comment:
If `start = 0` by default, the default constructor above seems not required.

Reply:
updated